k8s/aat/common/ia/aip-frontend.yaml

```
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-aip-frontend
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-aip-frontend
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: aip-frontend
+    version: 0.0.1
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/aip-frontend:prod-0d37a264
+      ingressHost: ia-aip-frontend-aks.aat.platform.hmcts.net
```

you don't need AKS for new apps, divorce was doing a parallel run at the time
```suggestion
      ingressHost: ia-aip-frontend.aat.platform.hmcts.net
```

---

Ok